### ISSUE TITLE
multiply symbols errors from: Win32Exception(const char* message)

### ISSUE THREAD

**Issue by coinoperated0428** [microsoft/bond#1129]
Getting multiply symbol definition errors that appear to be a result of multiple includes from file:
vcpkg\installed\x64-windows\include\bond\ext\grpc\win_thread_pool.h

`    namespace detail
    {
        BOND_NORETURN void Win32Exception(const char* message)
        {
            throw std::system_error{
                std::error_code{ static_cast<int>(::GetLastError()), std::system_category() },
                message };
        }

    } // namespace detail
`
I was able to correct by adding inline specifier to the header file code definition i.e.:
inline BOND_NORETURN void Win32Exception(const char* message)

I'm i missing something fundamental here?

**Comment by chwarr** [comment]
Nope, you're not missing anything. That function is indeed missing `inline`. Care to submit a PR with the fix?

**Comment by bhardwajs** [comment]
@coinoperated0428 - I just sent a PR to fix this issue.

I think there needs to be a PR to update `vcpkg` port to pick up the latest changes after this is merged. https://github.com/microsoft/vcpkg/tree/master/ports/bond

**Comment by chwarr** [comment]
Thanks for the PR! Looks good to me, with two minor things.
